doc/source/security/data-masking.rst

```diff
@@ -20,29 +24,74 @@ The following command installs the plugin:
 
     INSTALL PLUGIN data_masking SONAME 'data_masking.so';
 
-The data masking functions are the following:
+.. rubric:: Data Masking functions
+
+The data masking functions have the following categories:
+
+* General purpose
+
+* Special purpose
+
+* Generating Random Data with Defined characteristics
+
+* Using Dictionaries to Generate Random Data
+
+.. note::
+
+    MySQL terminal in interactive mode displays binary as hex. As a workaround, use the `CONVERT <https://dev.mysql.com/doc/refman/8.0/en/cast-functions.html#function_convert>`_ function to return a string or integer.
```

Comment:
This is fixed as of 8.0.21, so not sure if it worth mentioning it here, or maybe note that the broken behavior is specific to 8.0.19 and 8.0.20

Reply:
added a note for secure_file_priv.